### PR TITLE
Feat/add chart

### LINKED ISSUE #277: Adopt the helm chart?
Please see https://github.com/helm/charts/issues/21103
The helm stable repo is about to shut down.
Would you be interested in maintaining the chart here?
We have tooling like shown in https://github.com/helm/charts-repo-actions-demo to support a workflow for testing and releasing charts. Visibility on the [Helm Hub](https://hub.helm.sh/) is done through https://github.com/helm/hub
CC @scottrigby

### PR DESCRIPTION
**Pull request by mariusv**
fixes #277

### CODE REVIEW

**Comment by jtblin** [comment]
Thanks a lot @mariusv !!!

File: charts/kube2iam/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## Prerequisites
+  - Kubernetes 1.4+ with Beta APIs enabled
+
+## Add repos & install the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm repo add kube2iam https://buildkite.github.io/charts/
```
Comment (zhimsel) [inline charts/kube2iam/README.md:24]: Pretty sure this URL is not correct. That repo is for Buildkite, not this chart. Was Github Pages set up for this repo? Once it is, I believe this URL should be `https://<pages_name>.github.io/charts` (or something along those lines).

File: charts/kube2iam/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+$ helm repo update
+$ helm install stable/kube2iam --name my-release
```
Comment (zhimsel) [inline charts/kube2iam/README.md:26]: This should not be `stable`...

```
helm install kube2iam/kube2iam --name my-release
```

File: charts/kube2iam/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+## Prerequisites
+  - Kubernetes 1.4+ with Beta APIs enabled
+
+## Add repos & install the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm repo add kube2iam https://buildkite.github.io/charts/
+$ helm repo update
```
Comment (zhimsel) [inline charts/kube2iam/README.md:25]: Not necessary when adding a new repo.

**Review by zhimsel — COMMENTED** [review]
In order for the chart to be properly hosted, a Github Pages site needs to be created for this directory and some Github Actions need to be set up to package the chart. Please see [the original comment in the issue](https://github.com/jtblin/kube2iam/issues/277#issue-675581486) for more info.

**Pull request merged**